creator/ingest_runs/tasks/ingest_run.py

```diff
@@ -1,12 +1,16 @@
 import logging
+from pprint import pformat
 
 from django.conf import settings
+from django.core.exceptions import ValidationError
+import pandas
 
 from creator.decorators import task
+from creator.analyses.analyzer import extract_data
 from creator.files.models import FileType
+from creator.ingest_runs.genomic_data_loader import GenomicDataLoader
 from creator.ingest_runs.models import IngestRun
 
-
 logger = logging.getLogger(__name__)
 
```

Comment:
Not sure if you've experienced the same thing but I noticed in my validation branch that I needed to set the log level to INFO on my module logger in order to get the info statements from my task to show up in the logs. I think the default log level is set to WARN so the info statements get filtered out